packages/cosmwasm-stargate/src/modules/wasm/aminomessages.spec.ts

```diff
@@ -29,7 +29,7 @@ describe("AminoTypes", () => {
         wasmByteCode: fromBase64("WUVMTE9XIFNVQk1BUklORQ=="),
         instantiatePermission: undefined,
       };
-      const aminoMsg = new AminoTypes({ prefix: "cosmos", additions: createWasmAminoConverters() }).toAmino({
+      const aminoMsg = new AminoTypes({ ...createWasmAminoConverters() }).toAmino({
```

Comment:
You still have `{ ...createWasmAminoConverters() }` in many places in this file. This is unnecessary, right?